test/es6/regex-prototype-properties.js

```diff
@@ -79,5 +79,20 @@ tests = tests.concat(nonGenericPropertyNames.map(function (name) {
         }
     };
 }));
+tests = tests.concat([
+    {
+        name: "RegExp.prototype.toString should be generic",
+        body: function () {
+            var re = {
+                source: "source",
+                flags: "gi"
+            };
+
+            var string = RegExp.prototype.toString.call(re);

```

Comment:
We should use the string `undefined` for them in that case. I didn't add a test specifically for this, but added some verifying the function when the properties are `undefined` in addition to non-string properties and when they are overridden on a subclass (ee246cce09c70b33b195c631050419b2785d8874).